service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOControllerTest.java

```diff
@@ -97,11 +97,14 @@
     @Test
     void aboutToStartCallbackShouldPrepareCaseForCMOWhenNoCaseManagementOrder() {
         Map<String, Object> data = Map.of(
-            HEARING_DETAILS_KEY, createHearingBookingsFromInitialDate(now()),
+            HEARING_DETAILS_KEY, createHearingBookingsFromInitialDate(now().minusDays(3)),
```

Comment:
just for my clarification, do we need to take care of bank holidays in date calculation?

Reply:
not really, it's just about past date, so this can be any number of days